src/workerd/api/worker-rpc.h

```diff
@@ -554,6 +554,11 @@ class EntrypointsModule: public jsg::Object {
   EntrypointsModule() = default;
   EntrypointsModule(jsg::Lock&, const jsg::Url&) {}
 
+  void waitUntil(kj::Promise<void> promise) {
+    JSG_REQUIRE(IoContext::hasCurrent(), Error, "waitUntil requires an active request");
```

Comment:
You don't really need this line, `current()` will already throw a JSG exception if there's no active request.